module/Decision/view/decision/member/index.phtml

```diff
@@ -42,29 +72,21 @@ $this->headTitle($this->translate('Members')); ?>
                 <h4><?= $this->translate('Organs') ?></h4>
             </div>
         </a>
-        <a href="<?= $this->url('member/dreamspark') ?>" class="panel panel-image col-sm-3 col-xs-6">
-            <div class="panel-body">
-                <div class="glyphicon glyphicon-cd"></div>
-                <h4><?= $this->translate('Microsoft Imagine') ?></h4>
-            </div>
-        </a>
-        <a href="https://gewis.nl/susos/account/websitelogin.php" class="panel panel-image col-sm-3 col-xs-6">
-            <div class="panel-body">
-                <div class="glyphicon glyphicon-glass"></div>
-                <h4><?= $this->translate('SuSOS') ?></h4>
-            </div>
-        </a>
-        <a href="<?= $this->url('decision/search') ?>" class="panel panel-image col-sm-3 col-xs-6">
-            <div class="panel-body">
-                <div class="glyphicon glyphicon-search"></div>
-                <h4><?= $this->translate('Decisions') ?></h4>
-            </div>
-        </a>
+    </div>
+    
+    <h3 class="h-underline"><?= $this->translate('Active members') ?></h3>
```

Comment:
We probably should add an acl check to only show this to active members. You can look at how this is done for the admin button in the menu